README.md

```diff
@@ -49,7 +49,7 @@ A note about the gamer method. If you already know the gamers xuid you can use t
 | Method | Value | Optional | Short Description |
 |---       |---    |---              |---       |
 | `get(method=<string>)`        | string  | `term=<string>` | API calls. |
-| `send_message(message=<string>, xuids=[])` | string, list | n/a | Send a message to the following xuids |
+| `send_message(message=<string>, gamers=[])` | string, list | n/a | Send a message to the following xuids |
```

Comment:
Thanks for updating the code to change the references of `gamers` to `xuids`. Can you remove this commit since the commit isn't necessary anymore?

Should have only two commits, one for adding the additional api calls and one for changing the references in `send_message` method.